src/main/java/dk/kb/netarchivesuite/solrwayback/solr/SolrStats.java

```diff
@@ -0,0 +1,80 @@
+package dk.kb.netarchivesuite.solrwayback.solr;
+
+import com.google.gson.Gson;
+import org.apache.solr.client.solrj.SolrQuery;
+import org.apache.solr.client.solrj.response.QueryResponse;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Get stats through the solr <a href="https://solr.apache.org/guide/8_11/the-stats-component.html">stats component</a>.
+ * Methods in this class
+ */
+public class SolrStats {
```

Comment:
Consider adding support for `percentiles` - it is quite a nice feature for e.g. looking at image sizes.

Reply:
I have added a method for this now. Let me know what you think.

Reply:
Looks good!